src/lib/containers/home_page/programs/Programs.Desktop.tsx

```diff
@@ -19,7 +18,7 @@ export default function ProgramsDesktop({
         style={{ backgroundColor: color.replace(')', ',.05)') }}
       >
         <div className="Programs__Card__header__icon">
-          <Icon type={iconValue} iconOptions={iconOptions} />
+          <ImageFileHandle fileHandleAssociation={fileHandleAssociation!} />
```

Comment:
Can we handle the undefined case instead of asserting that it's defined? Not sure what happens if undefined gets passed into ImageFileHandle but it seems like it may be possible if there's a config error

Reply:
Also might indicate a loading state? Not sure